tests/unit/parser/test_unit_tests.py

```diff
@@ -173,3 +226,62 @@ def test_multiple_unit_tests(self):
         for unit_test in self.parser.manifest.unit_tests.values():
             self.assertEqual(len(unit_test.depends_on.nodes), 1)
             self.assertEqual(unit_test.depends_on.nodes[0], "model.snowplow.my_model")
+
+    def _parametrize_test_promote_non_none_row(
```

Comment:
took me a while to realize the purpose of this method wasn't just to parametrize / set up a test but also to run assertions. non-blocking naming suggestion would be to call this `_assert_fixture_yml_matches_expected_rows`, or something similar?

Reply:
(more examples in tests/unit/parser/test_manifest.py)